src/Modelling/StateDiagram/EnumArrows.hs

```diff
@@ -309,13 +307,13 @@ enumArrowsTask path task
       english "Consider the following state chart."
     case chartRenderer task of
        PlantUML
-         -> image $=<< liftIO $ drawSDToFile (combine path "plain") (hierarchicalSD task)
+         -> image $=<< liftIO $ PlantUML.drawSDToFile (combine path "plain/Diagram.svg") (hierarchicalSD task)
```

Comment:
This looks like a behavior change. Previously, the file would be written as `plainDiagram.svg`, not `plain/Diagram.svg`, right?

Reply:
You are right. I forgot that the original version just appended "Diagram.svg" (without prepending a "/"). I will fix it.